consensus/src/extension/election.rs

```diff
@@ -102,12 +103,13 @@ impl<U: UnitWithParents> CandidateElection<U> {
             return Ok(());
         }
         let relative_round = voter.round() - self.round;
+        let direct_parents = voter.direct_parents().cloned().collect::<Vec<_>>();
```

Comment:
Heh, most code we have uses `let direct_parents: Vec<_> = (...).collect()` rather than the turbofish.

Reply:
Wait, is the typing even necessary if these are only used in one call where the type is clear?